### ISSUE TITLE
How can I use report intermediate result about multi metrics?

### ISSUE THREAD

**Issue by coasxu** [microsoft/nni#4279]
How can I use report intermediate result about **multi metrics**?
For example, I want to report loss, train_loss and test_loss.
Thanks a lot

**Environment**:
- NNI version:
- Training service (local|remote|pai|aml|etc):
- Client OS:
- Server OS (for remote mode only): Ubuntu 16.04.6 LTS
- Python version: 
- PyTorch/TensorFlow version:
- Is conda/virtualenv/venv used?: Yes
- Is running in Docker?: No

**Configuration**:
 - Experiment config (remember to remove secrets!):
 - Search space:

**Log message**:
 - nnimanager.log:
 - dispatcher.log:
 - nnictl stdout and stderr:
 

**How to reproduce it?**:
